lib/field_struct/avro_schema/event.rb

```diff
@@ -36,6 +36,31 @@ def topic_name(value = :none)
           @topic_name || default_topic_name
         end
 
+        def default_schema_record_name
+          metadata.schema_name
+        end
+
+        def schema_record_name(value = :none)
+          @schema_record_name = value unless value == :none
+          @schema_record_name || default_schema_record_name
+        end
+
+        def default_schema_naming_strategy
+          :topic_name
```

Comment:
I think we should keep the default to the legacy version so we can safely upgrade the library without affecting the code. And then people can change the naming strategy manually once they are ready to migrate. Does that make sense?
```suggestion
          :legacy_topic_name
```

Reply:
This has been fixed @aemadrid